pkg/test/ginkgo/external.go

```diff
@@ -204,7 +204,9 @@ func extractBinaryFromReleaseImage(logger *log.Logger, releaseImageReferences *i
 }
 
 // runImageExtract extracts src from specified image to dst
-func runImageExtract(image, src, dst string, dockerConfigJsonPath string) error {
+func runImageExtract(image, src, dst string, dockerConfigJsonPath string, logger *log.Logger) error {
+	startTime := time.Now()
+	logger.Printf("Run image extract for release image %q at %v", image, startTime)
```

Comment:
Can you include the "src" in the logging string too? That'll tell us what we're trying to extract